be/src/column/nullable_column.cpp

```diff
@@ -113,7 +120,9 @@ ColumnPtr NullableColumn::replicate(const std::vector<uint32_t>& offsets) {
 }
 
 bool NullableColumn::append_nulls(size_t count) {
-    DCHECK_GT(count, 0u);
+    if (count == 0) {
```

Comment:
why check count in here?

Reply:
Otherwise every caller need to check it.

Reply:
I think caller check is also OK. This change will effect the exist caller performance.